lib/kitchen/provisioner/salt_solo.rb

```diff
@@ -216,6 +217,7 @@ def salt_command
         cmd << " --log-level=#{config[:log_level]}" if config[:log_level]
         cmd << " --id=#{config[:salt_minion_id]}" if config[:salt_minion_id]
         cmd << " test=#{config[:dry_run]}" if config[:dry_run]
+        cmd << " --force-color" if config[:salt_force_color] == 'true'
```

Comment:
should this `'true'` or `true`?

I would think you could just get rid of the == portion, since it should just be a boolean?

Reply:
Yes, you're right.
Sorry, I'm beginning with Ruby.

I'll fix that quickly

Reply:
no problem, thanks for updating it.  I was thinking about doing something like this as well, so thanks for submitting!

Daniel

Reply:
it's my pleasure to help. Thanks

Yann